src/functions/getConfiguration/framework/handler.ts

```diff
@@ -21,11 +21,18 @@ export async function handler(event: APIGatewayProxyEvent): Promise<Response> {
     return createResponse(msg, 500);
   }
 
+  const examinerRole = getExaminerRoleFromRequestContext(event.requestContext);
+  if (!examinerRole) {
+    const msg = 'No examiner role found in request context';
+    error(msg);
+    return createResponse(msg, 500);
```

Comment:
Is this an example of 5XX type of error? 
There is nothing wrong with "the server/service" itself - it's the request that is at fault here, not containing all the data (?) - 400 / 403 seems more applicable for me...

Reply:
I've wondered about this too. I originally put 500 as the response for when there's no staff number in the context as it's an issue with our service, although not necessarily this particular Lambda. Does 4xx make sense? There's nothing the client can alter about the request to make it work.

Reply:
Hmm... I see your point. 
For me, 5XX means that we failed at performing an operation even though the request was OK in the first place. 
I interpret this case similar to the scenario that you don't have a valid token - there is an underlining reason for this (eg. AD data not fully populated) but from the perspective of a client interacting with this service -> there is not enough data provided in the request to produce the response. 

@chrisnappin do you want to chime in on this? Seems like a judgment call =)

Reply:
Yes, if any context data is missing on the request it should be a 401 Not Authorised